azext_iot/digitaltwins/_help.py

```diff
@@ -113,6 +113,20 @@ def load_digitaltwins_help():
             az dt delete -n {instance_name} -y --no-wait
     """
 
+    helps["dt reset"] = """
+        type: command
+        short-summary: Reset an existing Digital Twins instance by deleting associated
+            dataplane assets. Currently only supports deleting models and twins.
+
+        examples:
+        - name: Reset all dataplane assets for a Digital Twins instance.
+          text: >
+            az dt reset -n {instance_name} --all
```

Comment:
Because we already have the `delete-all` commands for twins, their relationships, and models ...seems it would be appropriate to assume `az dt reset` would remove all data entities (twins + relationships, models).

What do you think?